docs/src/ComponentsPage.js

```diff
@@ -448,6 +448,14 @@ const ComponentsPage = React.createClass({
                   <ReactPlayground codeText={Samples.GridBasic} exampleClassName='bs-example-tabs' />
                 </div>
 
+
+                {/* Thumbnail */}
+                <div className='bs-docs-section'>
+                <h1 id='thumbnail' className='page-header'>Thumbnail</h1>
+                <p>Thumbnails are designed to showcase linked images with minimal required markup. You can extend the grid component with thumbnails.</p>
+                  <ReactPlayground codeText={Samples.Thumbnail} />
+                </div>
+
```

Comment:
Can you put this in the document in the same order as the navigation you used below?